docs/recipes/react.md

```diff
@@ -0,0 +1,124 @@
+# Testing React components
+
+## Setting up Babel
+
+The first thing you need to do is to set up `babel` to transpile JSX code from the tests. To do that, I'd recommend the [babelrc recipe](https://github.com/sindresorhus/ava/blob/master/docs/recipes/babelrc.md) using [`babel-preset-react`](http://babeljs.io/docs/plugins/preset-react/). You can also have a look at this [sample project config](https://github.com/adriantoine/ava-enzyme-demo)
+
+## Using [Enzyme](https://github.com/airbnb/enzyme/)
+
+Let's first see how to use AVA with one of the most popular React testing libraries: [Enzyme](https://github.com/airbnb/enzyme).
+
+If you only plan to use `shallow` component rendering, you don't need any extra setup.
+
+First install [Enzyme required packages](https://github.com/airbnb/enzyme/#installation):
+
+```console
+npm install --save-dev enzyme react-addons-test-utils react-dom
+```
+
+and you can use Enzyme straight away:
+
+```js
+import test from 'ava';
+import React from 'react';
+import {shallow} from 'enzyme';
+
+const Foo = ({children}) =>
+	<div className="Foo">
+		<span className="bar">bar</span>
+		{children}
+		<span className="bar">bar</span>
+	</div>;
+
+Foo.propTypes = {
+	children: React.PropTypes.any
+};
+
+test('has a .Foo class name', t => {
+	const wrapper = shallow(<Foo/>);
+	t.true(wrapper.hasClass('Foo'));
+});
+
+test('renders two `.Bar`', t => {
+	const wrapper = shallow(<Foo/>);
+	t.is(wrapper.find('.bar').length, 2);
+});
+
+test('renders children when passed in', t => {
+	const wrapper = shallow(
+		<Foo>
+			<div className="unique"/>
+		</Foo>
+	);
+	t.true(wrapper.contains(<div className="unique"/>));
+});
+```
+
+Enzyme also has a `mount` and `render` helper to test in an actual browser environment, if you want to use these helpers, you will have to setup a browser environment. To do so, you should check out the [browser testing recipe](https://github.com/sindresorhus/ava/blob/master/docs/recipes/browser-testing.md).
+
+To see an example of AVA working together with Enzyme set up for browser testing, you can have a look at [this sample project](https://github.com/adriantoine/ava-enzyme-demo).
+
+This is a basic example about how to integrate Enzyme with AVA, to get more information about using Enzyme for unit testing React component, please have a look at [Enzyme's documentation](http://airbnb.io/enzyme/).
```

Comment:
`with AVA, to get more information about` => `with AVA. For more information about`